src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  const adder = (value) => {
+    if (value === undefined) {
+      const temp = sum;
```

Comment:
what do you mean by temp? Looks like you should fix this naming

Reply:
my idea was `tmp` = temporary